logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/NewGroupConversationSystemMessagesCreator.kt

```diff
@@ -201,7 +202,7 @@ internal class NewGroupConversationSystemMessagesCreatorImpl(
     ): Either<CoreFailure, Unit> =
         persistMessage(
             Message.System(
-                uuid4().toString(),
+                LocalId.generate(),
```

Comment:
Not exactly related to this PR, but:

In the future we will need to agree with iOS and Web on how to generate IDs for system messages, if we want to backup and restore system messages as well without having duplicates.

In fact, this is something that we can already have issues with on Android.
When restoring a backup from client A onto client B, system messages that exist on both will be duplicated as we don't have a single ID for both.

Reply:
for backups we can ignore system messages not sure if those are needed anyway